core/domain/user_services.py

```diff
@@ -2008,9 +2008,6 @@ def get_community_reviewer_usernames(review_category, language_code=None):
             user_models.UserCommunityRightsModel
             .get_voiceover_reviewer_user_ids(language_code))
     elif review_category == constants.REVIEW_CATEGORY_QUESTION:
-        if language_code is not None:
```

Comment:
This isn't required for the question review!

Reply:
Wait -- why not? It's checking that the language code does *not* exist, which seems correct to me?

Reply:
The language code is only expected for the translation review. The question review is independent from language code! [i.e, the `language_code` property is unused in the `elif review_category == constants.REVIEW_CATEGORY_QUESTION:` section so we can avoid validating unused things!

[I'm fine with adding it, removed it as it's unwanted check!]

Reply:
Reverted!